client/Page/Admin/Chat/Index.tsx

```diff
@@ -2,7 +2,6 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import type { PageData } from '@/Types/types';
 import { AWPRootType } from '@/Types/types';
-import '@/assets/styles/app.css';
```

Comment:
We can't apply the shadow dom at this level?

Reply:
Done